src/Tizen.NUI/src/internal/PathConstrainer.cs

```diff
@@ -65,105 +56,11 @@ protected override void Dispose(DisposeTypes type)
             base.Dispose(type);
         }
 
-
-
-        /// <since_tizen> 3 </since_tizen>
-        public class Property : global::System.IDisposable
+        internal class Property
```

Comment:
please make this HiddenAPI (not internal) ; please add [EditorBrowsable], [Obsolete] attributes like others done.